libevmasm/Assembly.cpp

```diff
@@ -222,17 +222,21 @@ string Assembly::assemblyString(
 	return tmp.str();
 }
 
-Json::Value Assembly::createJsonValue(string _name, int _source, int _begin, int _end, string _value, string _jumpType)
+Json::Value Assembly::createJsonValue(string const& _name, AssemblyItem const& _item, std::string const& _value)
 {
 	Json::Value value{Json::objectValue};
 	value["name"] = _name;
-	value["source"] = _source;
-	value["begin"] = _begin;
-	value["end"] = _end;
+	value["begin"] = _item.location().start;
+	value["end"] = _item.location().end;
+	if (_item.m_modifierDepth != 0)
+		value["modifierDepth"] = static_cast<int>(_item.m_modifierDepth);
 	if (!_value.empty())
 		value["value"] = _value;
-	if (!_jumpType.empty())
-		value["jumpType"] = _jumpType;
```

Comment:
Yes, I removed `jumpType` yesterday, because I saw that it is just stored in `value` (for `Operation`). Regarding the old code, `jumpType` is only set for `PUSH`. Not sure why.

Reply:
@chriseth why is `jumpType` needed for `PUSH`?

Reply:
Or is it just a typo in the code?

Reply:
I don't see a problem with the old code. Yes, jumptype was output for push, but also for operation.

Reply:
I guess nobody really used it yet. Decision from call: store jump type's always in `jumpType` node.